examples/src/main/java/jaxrs/examples/multipart/MultipartResource.java

```diff
@@ -75,7 +75,7 @@ public Response applyForJob(@FormParam("name") String name,
 
         if (resumeFileName.toLowerCase().endsWith(".pdf")) {
             processPdfResume(resume.getContent());
-        } else {
+        //        } else {
```

Comment:
I'm wondering if we should either disable the checkstyle rule for this or add comments to ignore this line. WDYT?